From ff94067974825c224ac2bbdb0bc87c21b2a8985a Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Tue, 6 Nov 2018 09:49:20 +0800
Subject: [PATCH] arm64: dts: rockchip: Add PCIe node for rk1808 evb

Add PCIe node for rk1808 evb. If actviating PCIe, we
should set usbdrd_dwc3 as disabled.

Change-Id: I807e16d7950cbf76ceb07053c12905ee4566bdaa
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index e739be7e278f..c4daea857609 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -598,6 +598,12 @@
 	status = "okay";
 };
 
+&pcie0 {
+	reset-gpios = <&gpio0 RK_PB6 GPIO_ACTIVE_HIGH>;
+	/* Disable usbdrd_dwc3 simultaneouslly if using pcie0 */
+	status = "disabled";
+};
+
 &power {
 	pd_npu-supply = <&vdd_npu>;
 };
-- 
2.35.3

